src/components/Books.css

```diff
@@ -0,0 +1,29 @@
+.Books_form {
+  max-width: 700px;
+  margin: 0 auto;
+}
+
+.Books_form-input {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+}
```

Comment:
- [ ] It would be so much better if you follow the same color and padding for the books form. 

Your design

![Capture](https://user-images.githubusercontent.com/64173189/135726982-5d2c6cb5-1de2-4593-8ef4-284fb4e26f23.JPG)

The original design 

![Capture-1](https://user-images.githubusercontent.com/64173189/135726988-828b4c04-fea1-4a63-be10-f16d50bec9f5.JPG)

Please fix this minor issue 🥇